src/Pages/Home.js

```diff
@@ -1,51 +1,72 @@
-import { useContext } from "react";
 import styled from "styled-components";
-import { AuthenticationContext } from "../Contexts/AuthenticationContenxt";
+import { Link } from "react-router-dom";
 
 function Home() {
-  const { user, dispatch } = useContext(AuthenticationContext);
-
-  const onClickHandler = () => {
-    localStorage.removeItem("user");
-    dispatch({ type: "SIGNOUT" });
-  };
-
   return (
     <HomeContainer>
-      <h1>{`Welcome ${user.email}`}</h1>
-      <StyledButton type="button" onClick={onClickHandler}>
-        Signout
-      </StyledButton>
+      <StyledList>
+        {Array(20)
+          .fill()
+          .map((_, index) => (
+            <li key={Math.random()}>
```

Comment:
the keys of elements in an element list must never change. They have to be constant over all render cycles of your app. The way you use it here reassigns a new random value in every render cycle. Please find a better way for this.

Reply:
Done

Reply:
This are just placeholders, next step will be the creation and saving of new notes to the db.